playwright-e2e/dsm/enums.ts

```diff
@@ -26,22 +26,47 @@ export enum DataFilter {
 // Table column headers
 
 export enum CustomizeView {
-  CONTACT_INFORMATION = 'Contact Information Columns',
```

Comment:
this is just changing things around in this enum to make it alphabetical - to make it easier to spot a missing CustomizeView column group